WordPress/openverse-frontend
# Add Checkbox component

## Fixes


Fixes #405 by @obulat

## Description


This PR creates a standalone `Checkbox` component with the new styles, and uses it throughout the app:
<img width="330" alt="Screen Shot 2021-11-09 at 5 53 12 PM" src="https://user-images.githubusercontent.com/15233243/140947040-646fd39b-1bf7-41e9-9ee6-9348b4101b08.png">
The last checkbox in the screenshot is disabled. I couldn't find the actual disabled variant in Figma, @panchovm .

## Testing Instructions

Checkout the branch and run `npm run storybook` to see the Checkbox story there, and verify that it works both with a string label, and a `License` component label.
Then, run `npm run dev` and test the Checkboxes in use on the home page (license types inputs) and the filter sidebar. Check/uncheck them to see that the filters are applied, and new media is fetched.
`npm run test` should also run without problems :)
## Checklist


[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
